federator/src/contracts/IFederationV3.ts

```diff
@@ -62,21 +62,20 @@ export class IFederationV3 {
     return this.federationContract.options.address;
   }
 
-  getPastEvents(eventName: string, options: any, from: string): Promise<any> {
-    return this.federationContract.getPastEvents(eventName, options).call({ from });
+  getPastEvents(eventName: string, options: any): Promise<[any]> {
+    return this.federationContract.getPastEvents(eventName, options);
```

Comment:
why did we remove the from?

Reply:
Form is not necesary for getting past events as it is not a .call() it actually calls to eth_getLogs

Reply:
The getPastEvents is directly a call, so it doesn't need to pass the `from` address

Reply:
awesome!